test/cloudwatch-adapter.spec.js

```diff
@@ -246,9 +249,15 @@ describe('cloudwatch adapter', function() {
                 expect(point.namespace).to.equal(`AWS/${point.product}`);
             }
 
-            expect(_.keys(expected_metrics[point.namespace])).to.include(point.name);
-            expect(statistics).to.include(point.statistic);
-            expect(point.units).to.equal(expected_metrics[point.namespace][point.name].units);
+            // We allow metrics not in the above set--Cloudwatch adds
+            // new metrics from time to time, so those shouldn't
+            // result in test failures. We do log a warning, though.
+            if (! _.contains(_.keys(expected_metrics[point.namespace]), point.name)) {
+                console.log(`Possible new Cloudwatch Metric ${point.name}. Consider adding to set of supported metrics.`);
```

Comment:
Calling this a "Possible new Cloudwatch Metric" implies that there's a chance it's something else which seems confusing. How about we just say "Unexpected Cloudwatch Metric...".